content/en/serverless/installation/go.md

```diff
@@ -24,14 +24,64 @@ aliases:
 
 If your Go Lambda functions are still using runtime `go1.x`, you must either [migrate][1] to `provided.al2` or use the [Datadog Forwarder][2] instead of the Datadog Lambda Extension.
 
-### Install the Datadog Lambda library
+{{< tabs >}}
+{{% tab "Datadog CLI" %}}
 
-Install the [Datadog Lambda library][3] locally by running the following command:
+The Datadog CLI modifies existing Lambda functions' configurations to enable instrumentation without requiring a new deployment. It is the quickest way to get started with Datadog's serverless monitoring.
+
+You can also add the command to your CI/CD pipelines to enable instrumentation for all your serverless applications. Run the command _after_ your normal serverless application deployment, so that changes made by the Datadog CLI command are not overridden.
```

Comment:
```suggestion
You can also add the [instrumentation command](#instrumentation) to your CI/CD pipelines to enable instrumentation for all your serverless applications. Run the command _after_ your normal serverless application deployment, so that changes made by the Datadog CLI command are not overridden.
```